samples/kvsWebRTCClientMaster.c

```diff
@@ -313,8 +318,14 @@ PVOID sendAudioPackets(PVOID args)
         MUTEX_LOCK(pSampleConfiguration->sampleConfigurationObjLock);
         for (i = 0; i < pSampleConfiguration->streamingSessionCount; ++i) {
             status = writeFrame(pSampleConfiguration->sampleStreamingSessionList[i]->pAudioRtcRtpTransceiver, &frame);
-            if (status != STATUS_SUCCESS) {
+            if (status != STATUS_SUCCESS && status != STATUS_SRTP_NOT_READY_YET) {
                 printf("writeFrame failed with 0x%08x", status);
```

Comment:
BTW, we might also use the iffef whether we print or not

Reply:
Done